src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -86,11 +86,13 @@ object Task {
     //scalastyle:off cyclomatic.complexity method.length
     override def update(update: TaskStateOp): TaskStateChange = update match {
       // case 1: now running
-      case TaskStateOp.MesosUpdate(_, MarathonTaskStatus.Running(mesosStatus), now) if !hasStartedRunning =>
+      case TaskStateOp.MesosUpdate(_, taskStatus: MarathonTaskStatus.Running, now) if !hasStartedRunning =>
         val updated = copy(
+          networking = networking.copy(networkInfos = taskStatus.networkInfos),
           status = status.copy(
             startedAt = Some(now),
-            mesosStatus = mesosStatus))
+            mesosStatus = taskStatus.mesosStatus)
```

Comment:
aren't we essentially storing the network infos twice in our task? Once as part of the status and once as part of the networking? same potentially in the protobuf?

Or am I misunderstanding this?